subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan.c

```diff
@@ -467,10 +468,9 @@ static void isr_tx(void *param)
 #endif /* CONFIG_BT_CTLR_GPIO_LNA_PIN */
 }
 
-static void isr_done(void *param)
+static void isr_common_done(void *param)
 {
 	struct node_rx_pdu *node_rx;
-	u32_t start_us;
 
 	/* TODO: MOVE to a common interface, isr_lll_radio_status? */
```

Comment:
Does this fulfill the TODO?

Reply:
No, todo is more of a clean up task, to find common code patterns and convert them to reusable functions.